tests/storage/lvm_test.py

```diff
@@ -1090,6 +1090,95 @@ def test_vg_get_vgs_with_non_existing_vg(fake_devices):
     assert len(vgs) == 0
 
 
+def test_vg_get_all_miss_all_stale_but_one(fake_devices):
```

Comment:
This is already tested in
- https://github.com/oVirt/vdsm/blob/013692737dc9dd89b977cb8a91e479c8757fd0db/tests/storage/lvm_test.py#L1529
- https://github.com/oVirt/vdsm/blob/013692737dc9dd89b977cb8a91e479c8757fd0db/tests/storage/lvm_test.py#L1547
- https://github.com/oVirt/vdsm/blob/013692737dc9dd89b977cb8a91e479c8757fd0db/tests/storage/lvm_test.py#L1564
- https://github.com/oVirt/vdsm/blob/013692737dc9dd89b977cb8a91e479c8757fd0db/tests/storage/lvm_test.py#L1564

We may not need any new tests.

Reply:
Right, I have removed them.